examples/202-NN_examples.ipynb

```diff
@@ -42,7 +42,7 @@
   },
```

Comment:
<p>I guess we need to add TFT here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/etna-team/etna/pull/268/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>